README.md

```diff
@@ -732,6 +729,48 @@ manifest.ms = {
 };
 ```
 
+## Generating Icons
+
+Managing all the various icon sizes and types can be overwhelming. One solution is to start with a base image which can be 
+use to generate the necessary icon permutations for your environment.  You can use [ember-cli-image-transformer](https://github.com/jrjohnson/ember-cli-image-transformer) to handle this task.
+
+If your `manifest.js` needs the icons 
+
+```javascript
+"icons": [
+    {
+      "src": "/images/icons/android-chrome-192.png",
+      "sizes": "192x192",
+      "type":"image/png"
+    },
+    {
+      "src": "/images/icons/android-chrome-512.png",
+      "sizes": "512x512",
+      "type": "image/png"
+    }
+  ]
+  ```
```

Comment:
Just an idea here, maybe we can use the same size array that you use in the other config. I like to give users ideas on how to make the config a bit more DRY 😄 

```javascript
// config/manifest.js
export default function() {
  return {
    icons: [192, 512].map((size) => ({
      src: `/images/icons/android-chrome-${size}.png`,
      sizes: `${size}`,
      type: "image/png"
    }))
  };
}
```

Reply:
Thanks for this especially. I'm 100% going to steal it to improve our `manifest.js`!